### PR TITLE
Add #59797 to freeze branch

### PR DESCRIPTION
**Pull request by Ch3LL**
Add #59797 to freeze branch.

### CODE REVIEW

**Review by garethgreenaway — APPROVED** [review]

File: tests/pytests/integration/modules/test_pillar.py [hunk 1]
```diff
@@ -478,3 +478,118 @@ def test_pillar_refresh_pillar_scheduler(salt_cli, salt_minion):
 
     # Refresh pillar once we're done
     salt_cli.run("saltutil.refresh_pillar", wait=True, minion_tgt=salt_minion.id)
+
+
+@pytest.mark.slow_test
+def test_pillar_match_filter_by_minion_id(salt_cli, salt_minion):
```
Comment (s0undt3ch) [inline tests/pytests/integration/modules/test_pillar.py:484]: These are actually 3 tests, which we can parametrize with pytest.

Comment (s0undt3ch) [reply tests/pytests/integration/modules/test_pillar.py:484]: I'm cool merging like this on the freeze branch, we should fix on the master branch though

**Review by s0undt3ch — APPROVED** [review]

File: tests/pytests/integration/modules/test_pillar.py (hunk 1 above)
Comment (Ch3LL) [reply tests/pytests/integration/modules/test_pillar.py:484]: K I'll leave a not on the master PR

Comment (Ch3LL) [reply tests/pytests/integration/modules/test_pillar.py:484]: Nevermind, its already merged. I'll prep a PR for master.

**Review by krionbsd — APPROVED** [review]

**Pull request merged**